examples/metadata_extraction/README.md

```diff
@@ -0,0 +1,89 @@
+
+# Metadata Extraction from Research Papers
+
+## Objective
+
+This use case demonstrates how Aggrag can be used to extract metadata from research papers. The objective is to extract metadata **strictly** in the following format: 
+
+{
+"title":string, 
+"authors":[list of strings],
+"organizations":[list of strings],
+"keywords":[list of strings]
+}
+
+
+## Usecase-Iteration Flow
+
+Aggrag follows a Usecase-Iteration flow, where each use case consists of multiple iterations. Each iteration is designed to improve upon the previous one, either by refining the prompt, adjusting the RAG parameters, or incorporating additional data or techniques.
+
+An integral part of an iteration is setting up the evaluation model. Although more experiments are to be done, intuitivally it makes sense to evaluate all iterations on same or similar metrics. 
+
+In this example, we are evaluating generated responses' strict adherence to metadata schema and the time taken to extract. 
+
+Note: ideally, adding more than one paper should not affect the time taken as aggrag library is designed to work asynchronously. But in the current example iterations, we are noticing some latency being added as more pdf papers are added. 
+
+## Testing the Flow
+
+You can get your hands dirty by setting up an example flow to learn the capabilities of what  the aggrag example flows by following these steps:
+
+1. Setup and run the aggrag project locally. Steps here: `https://github.com/garvk/aggrag#local-setup`
+2. Create a new 'use-case' on aggrag UI.
+3. On iteration 1, import a sample cforge file located in examples/ directory. 'Import' is button on top-right banner on the UI.
+4. To run the flow locally, you'll need to set up the aggrag environment and provide your model API keys in the .env file. Follow the instructions in the Local Setup section of the README.
+
```

Comment:
Should we also mention that one can follow the provided "hints" to create a new iteration.

Reply:
Yes, adding.